packages/react/src/components/LightboxMediaViewer/LightboxMediaViewer.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ExpressiveModal } from '../ExpressiveModal';
+import { ModalBody } from 'carbon-components-react';
+import { settings } from 'carbon-components';
+import { DDS_LIGHTBOX_MEDIA_VIEWER } from '../../internal/FeatureFlags';
+import {
+  featureFlag,
+  settings as ddsSettings,
+} from '@carbon/ibmdotcom-utilities';
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * LightboxMediaViewer Component
+ *
+ * @param {object} props props object
+ * @param {string} props.title LightboxMediaViewer media title
+ * @param {string} props.copy LightboxMediaViewer media short description
+ * @param {string} props.image LightboxMediaViewer responsive image object
+ * @param {boolean} props.open sets whether the modal is open/close
+ * @param {Function} props.onClose do something on close in addition, return false to completely replace
+ * @returns {*} JSX Object
+ */
+const LightboxMediaViewer = ({ title, copy, image, open, onClose }) => {
+  /**
+   * Set default image
+   *
+   * @param {*} image responsive image object
+   * @returns {string} default image url
+   */
+  const defaultImage = image => {
+    return image.uri.md
+      ? image.uri.md
+      : image.uri.lg
+      ? image.uri.lg
+      : image.uri.sm;
+  };
+
+  if (!image.uri.lg) {
+    return null;
+  }
+  return featureFlag(
+    DDS_LIGHTBOX_MEDIA_VIEWER,
+    <section
+      data-autoid={`${stablePrefix}--lightbox-media-viewer`}
+      className={`${prefix}--lightbox-media-viewer`}>
+      <ExpressiveModal open={open} fullwidth={true} onClose={onClose}>
+        <ModalBody>
+          <div className={`${prefix}--lightbox-media-viewer__container`}>
+            <div className={`${prefix}--lightbox-media-viewer__row`}>
+              <div
+                data-autoid={`${stablePrefix}--lightbox-media-viewer__image`}
+                className={`${prefix}--lightbox-media-viewer__image`}>
+                <picture>
+                  {image.uri.lg && (
+                    <source
+                      media="(min-width: 672px)"
+                      srcSet={`${image.uri.lg}`}
+                    />
+                  )}
+                  {image.uri.md && (
+                    <source
+                      media="(min-width: 320px)"
+                      srcSet={`${image.uri.md}`}
+                    />
+                  )}
+                  {image.uri.sm && (
+                    <source
+                      media="(max-width: 320px)"
+                      srcSet={`${image.uri.sm}`}
+                    />
+                  )}
+                  <img src={defaultImage(image)} alt={image.alt}></img>
+                </picture>
```

Comment:
While this work might still be worked on I'm pretty sure it'll just be an array of image objects. `[{ src: '', alt: '' }, { src: '', alt: '' }]`, and this component for the time being only needs to take in one image. The object and logic you have above isn't really necessary at this time, especially with the responsive image coming down the road soon.